CMakeLists.txt

```diff
@@ -301,6 +301,9 @@ if(CPPDAP_BUILD_TESTS)
         list(APPEND DAP_TEST_LIST
             ${CPPDAP_GOOGLETEST_DIR}/googletest/src/gtest-all.cc
         )
+	set_property(SOURCE
+            ${CPPDAP_GOOGLETEST_DIR}/googletest/src/gtest-all.cc
+            APPEND PROPERTY COMPILE_OPTIONS -w)
```

Comment:
Incorrect indentation. But more importantly, I don't think this will work on MSVC builds. I think the problem here is the fact that we're adding `gtest-all.cc` to cppdap's `SOURCE` var, so it inherits the compilation flags from it. We can see that when `CPPDAP_USE_EXTERNAL_GTEST_PACKAGE` is defined, we use `find_package`, which brings in the `GTest::gtest` target. I think in the `else()` case above, we could use `add_subdirectory` to add `${CPPDAP_GOOGLETEST_DIR}/googletest`, which would allow us to just link against the same target below. This would no longer build with the same compilation/warning flags.

Reply:
ok, looking at that.

Reply:
MSVC allows `-w` to disable warnings:
- Options are specified by either a forward slash (/) or a dash (-).  https://learn.microsoft.com/en-us/cpp/build/reference/compiler-command-line-syntax?view=msvc-170#:~:text=Options%20are%20specified%20by%20either%20a%20forward%20slash%20(/)%20or%20a%20dash%20(%2D). 
- `/w` disables warnings: https://learn.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=msvc-170